src/lib/client/utils/loadClerkJsScript.ts

```diff
@@ -0,0 +1,41 @@
+import { parsePublishableKey } from '@clerk/shared/keys'
+import { loadScript } from '@clerk/shared/loadScript'
+
+import type { ClerkInitOptions } from './types'
+
+const FAILED_TO_LOAD_ERROR = 'Clerk: Failed to load Clerk'
+
+export async function loadClerkJsScript(opts: ClerkInitOptions) {
+	const { publishableKey } = opts
```

Comment:
Found this technique [here](https://github.com/clerk/javascript/blob/8488bac8238e02d68f4c1025168b40a6101e29cc/packages/react/src/utils/loadClerkJsScript.ts#L18)